runtime-config-linux.md

```diff
@@ -180,7 +180,14 @@ For more information, see [the memory cgroup man page](https://www.kernel.org/do
 
 #### Set oom_score_adj
 
-More information on `oom_score_adj` available [here](https://www.kernel.org/doc/Documentation/filesystems/proc.txt).
+oom_score_adj sets heuristic regarding how the process is evaluated by the kernel during memory pressure.
```

Comment:
On Tue, Jan 05, 2016 at 12:47:42PM -0800, Vincent Batts wrote:
> but it is per process

Right.  I just think “sets heuristic regarding how the process is
evaluated by the kernel” could be condensed without losing
information.  How about something closer to the kernel's §3.1 lead in
[1]:

  `oom_score_adj` adjusts the OOM-killer badness heuristic for this
  process.

?

[1]: https://www.kernel.org/doc/Documentation/filesystems/proc.txt